workloads/hyper-scale/common.sh

```diff
@@ -156,6 +171,11 @@ cleanup(){
     sleep 10
     ROUTE_ID=$(aws route53 list-hosted-zones --output text --query HostedZones | grep $BASEDOMAIN | grep -v terraform | awk '{print$2}' | awk -F/ '{print$3}')
     for id in $ROUTE_ID; do aws route53 delete-hosted-zone --id=$id || true ; done
+    rm -f *-admin-kubeconfig || true
+    rm -f pull-secret || true
+    rm -rf  kube-burner.tar.gz|| true
```

Comment:
there is an extra space between rm -rf and kube-burner

Reply:
Thanks for the catch, updated